backend/src/openarchiefbeheer/destruction/models.py

```diff
@@ -52,6 +52,13 @@ class DestructionList(models.Model):
         verbose_name=_("author"),
         help_text=_("Creator of destruction list."),
     )
+    comment = models.TextField(
+        _("comment"),
+        max_length=2000,
+        blank=True,
+        null=True,
+        help_text=_("Explanation of the destruction list."),
+    )
```

Comment:
```suggestion
    comment = models.TextField(
        _("comment"),
        max_length=2000,
        blank=True,
        help_text=_("Explanation of the destruction list."),
    )
```
See https://docs.djangoproject.com/en/5.1/ref/models/fields/#null

Maybe "description" would be better as a name for this field?
Also: note that the max_length attribute is not enforced at DB level for TextFields (https://docs.djangoproject.com/en/5.1/ref/models/fields/#django.db.models.TextField). It is only used in widgets, but since we are not using widgets, I would remove it.